src/libs/vue-h-zoom.js

```diff
@@ -1,3 +1,11 @@
+const DEFAULT_BACKGROUND_OPTIONS = {
+  image: 'none',
+  color: '#fff',
+  repeat: false,
+  contain: true,
+  position: 'top left'
```

Comment:
what is this value represent?

Reply:
the will be the position of the 'custom background', will be used in custom background style
```javascript
    backgroundImgStyle: function () {
      return {
        image: this.backgroundOption.image || DEFAULT_BACKGROUND_OPTIONS.image,
        repeat: this.backgroundOption.repeat ? 'repeat' : 'no-repeat',
        color: this.backgroundOption.color || DEFAULT_BACKGROUND_OPTIONS.color,
        contain: this.backgroundOption.contain ? 'contain' : 'cover',
        position: this.backgroundOption.position || DEFAULT_BACKGROUND_OPTIONS.position
      }
    },
```